qa/qa-captain/bashrc

```diff
@@ -0,0 +1,1269 @@
+#etcd-s3-secret-key: "YOUR_SECRET_KEY"
+
+##DUALSTACK
+#cluster-cidr: 10.42.0.0/16,2001:cafe:42:0::/56
+#service-cidr: 10.43.0.0/16,2001:cafe:42:1::/112
+
+#flannel-ipv6-masq: true
+#disable: rke2-ingress-nginx
+
+## SPLIT ROLES
+
+##--etcd-only---
+#disable-apiserver: true
+#disable-controller-manager: true
+#disable-scheduler: true
+#node-taint:
+#  - node-role.kubernetes.io/etcd:NoExecute
+
+##--etcd-cp---
+#node-taint:
+#  - node-role.kubernetes.io/control-plane:NoSchedule
+#  - node-role.kubernetes.io/etcd:NoExecute
+
+##--etcd-worker---
+#disable-apiserver: true
+#disable-controller-manager: true
+#disable-scheduler: true
+
+##--cp-only---
+#disable-etcd: true
+#node-taint:
+#  - node-role.kubernetes.io/control-plane:NoSchedule
+
+##--cp-worker---
+#disable-etcd: true
+
+EOF
+    sudo cp "${_product}"-config.yaml /etc/rancher/"${_product}"/config.yaml;
+}
+
+# --- set registries to look for bci hardened images ---
+set_registries() {
+    cat <<'EOF' >> registries.yaml
+    mirrors:
+    docker.io:
+        rewrite:
+        "^rancher/hardened-etcd(.*)": "bcibase/hardened-etcd$1"
+        "^rancher/hardened-kubernetes(.*)": "bcibase/hardened-kubernetes$1"
+        "^rancher/hardened-rke2-runtime(.*)": "bcibase/hardened-rke2-runtime$1"
+        "^rancher/nginx-ingress-controller(.*)": "bcibase/nginx-ingress-controller$1"
+        "^rancher/nginx-ingress-controller-chroot(.*)": "bcibase/nginx-ingress-controller-chroot$1"
+        "^rancher/hardened-calico(.*)": "bcibase/hardened-calico$1"
+        "^rancher/hardened-sriov-network-resources-injector(.*)": "bcibase/hardened-sriov-network-resources-injector$1"
+        "^rancher/hardened-k8s-metrics-server(.*)": "bcibase/hardened-k8s-metrics-server$1"
+        "^rancher/hardened-sriov-network-webhook(.*)": "bcibase/hardened-sriov-network-webhook$1"
+        "^rancher/hardened-sriov-network-operator(.*)": "bcibase/hardened-sriov-network-operator$1"
+        "^rancher/hardened-sriov-network-device-plugin(.*)": "bcibase/hardened-sriov-network-device-plugin$1"
+        "^rancher/hardened-flannel(.*)": "bcibase/hardened-flannel$1"
+        "^rancher/hardened-crictl(.*)": "bcibase/hardened-crictl$1"
+        "^rancher/hardened-ib-sriov-cni(.*)": "bcibase/hardened-ib-sriov-cni$1"
+        "^rancher/hardened-runc(.*)": "bcibase/hardened-runc$1"
+        "^rancher/hardened-rke2-cloud-provider(.*)": "bcibase/hardened-rke2-cloud-provider$1"
+        "^rancher/hardened-cni-plugins(.*)": "bcibase/hardened-cni-plugins$1"
+        "^rancher/hardened-dns-node-cache(.*)": "bcibase/hardened-dns-node-cache$1"
+        "^rancher/hardened-containerd(.*)": "bcibase/hardened-containerd$1"
+        "^rancher/hardened-cluster-autoscaler(.*)": "bcibase/hardened-cluster-autoscaler$1"
+        "^rancher/hardened-coredns(.*)": "bcibase/hardened-coredns$1"
+        "^rancher/hardened-multus-cni(.*)": "bcibase/hardened-multus-cni$1"
+        "^rancher/hardened-whereabouts(.*)": "bcibase/hardened-whereabouts$1"
```

Comment:
what is bcibase? Is this an internal mirror somewhere? It doesn't appear to be getting any updates on Dockerhub.

Reply:
I missed this in my initial review. There should not be any reference to the bcibase repositories. This is not to be used for anything. Great catch @matttrach

Reply:
Yeah thanks for catching us out skimming 😬 @VestigeJ Can we update this to just setup a basic registries.yaml file now without the bcibase-specific testing stuff?

Reply:
Pulled the function that checks for images built with bci